src/main/java/net/tammon/sip/packets/ReadOnlyDataResponse.java

```diff
@@ -34,9 +34,14 @@
 import java.io.InvalidClassException;
 import java.util.Arrays;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class ReadOnlyDataResponse extends AbstractPacket implements Response {
 
-    private final static int messageType = 72;
+    private static Logger _logger = LoggerFactory.getLogger("net.tammon.sip");
+    
+    private final static int messageType = ReadOnlyData.MSG_READ_ONLY_DATA+1;
```

Comment:
Be consistent in message type id naming
```suggestion
    private final static int MSG_TYPE_ID = ReadOnlyData.MSG_TYPE_ID+1;
```